### ISSUE TITLE
Dead?

### ISSUE THREAD

**Issue by quattro** [limix/limix#116]
Is this project dead? I've found it useful for simulations in my group, but it hasn't managed to merge contributed fixes into a release, let alone provide fixes.

It's fine if it is, maintaining software is challenging in an academic setting, but at least flagging it for users to know not to import it would be helpful.

**Comment by ManuelSokolov** [comment]
Indeed, this package was very easy to use and faster then any other alternatives. Adapting code to statsmodels makes code take more 8 or 9 times than when limix was used. Is there any alternative recommended ?
